Add pagination for CurrentUser::guilds

NilsIrl opened the pull request.

NilsIrl committed:
Add pagination for CurrentUser::guilds

arqunis commented on src/model/user.rs:
````diff
@@ -166,7 +166,19 @@ impl CurrentUser {
     /// ```
     #[inline]
````
```suggestion

```
This should no longer be marked as `#[inline]`. I fear it would cause more harm than good.
It's a beneficial attribute for small functions, as it permits them to be inlined across crate boundaries. But if they're longer than three statements, then I believe it shouldn't apply to them anymore.

NilsIrl replied:
Fixed

NilsIrl committed:
Removing inlining of CurrentUser::guilds

Now that CurrentUser::guilds is larger, inlining probably causes more harm good

Co-authored-by: Alex M. M. <acdenissk69@gmail.com>
